torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7878,6 +7909,16 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            check_batched_gradgrad=False,
            supports_forward_ad=True,
            decorators=[skipCUDAIfNoMagma, skipCUDAIfRocm, skipCPUIfNoLapack]),
+    OpInfo('linalg.solve_triangular',
+           aten_name='linalg_solve_triangular',
+           op=torch.linalg.solve_triangular,
+           dtypes=floating_and_complex_types(),
+           sample_inputs_func=sample_inputs_linalg_solve_triangular,
+           # linalg.solve_triangular cannot be batched over because of a call to out.copy_(result);
+           check_batched_grad=False,
+           check_batched_gradgrad=False,
+           supports_forward_ad=True,
+           decorators=[skipCUDAIfRocm, skipCPUIfNoLapack]),
```

Comment:
Is `skipCUDAIfRocm` necessary here?

Reply:
Most probably not anymore! I'll remove it.

Reply:
Even more, I believe both the decorators can be removed.

Reply:
Interestingly enough, we use magma for `solve_triangular`, but just for huge matrices, so I don't think it's necessary to skip it here, right? It's not used in this test.

Reply:
That's interesting that there is no `skipCPUIfNoBlas`.
I agree we don't need the skip magma decorator here because inputs are too small.